libraries/AP_Mount/AP_Mount_Backend.cpp

```diff
@@ -7,6 +7,12 @@ extern const AP_HAL::HAL& hal;
 
 #define AP_MOUNT_UPDATE_DT 0.02     // update rate in seconds.  update() should be called at this rate
 
+// Default init function for every mount
+void AP_Mount_Backend::init(){
```

Comment:
small formatting issue here is that we normally put the curly bracket on the next line.